logcall/__openerp__.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```

Comment:
If this is longstanding module, just let it be not to break any compatibility for migration.